00.md

```diff
@@ -41,7 +41,7 @@ KeyPackages are typically consumed when joining groups. To handle race condition
   "kind": 443,
   "created_at": 1693876543,
   "pubkey": "f1e2d3c4b5a697887766554433221100ffeeddccbbaa99887766554433221100",
-  "content": "0123456789abcdef...",
+  "content": "v1:SGVsbG8xMjM0NTY3ODlhYmNkZWY...",
```

Comment:
I think it's better to use the event tags to specified this new version rather than a prefix